Sources/Client/Model/User/User.swift

```diff
@@ -78,6 +79,9 @@ public struct User: Codable {
     public internal(set) var currentDevice: Device?
     /// Muted users.
     public internal(set) var mutedUsers: [MutedUser]
+    /// Teams the user belongs to. You need to enable multi-tenancy if you want to use this, else it'll be nil.
+    /// Refer to [docs](https://getstream.io/chat/docs/multi_tenant_chat/?language=swift) for more info.
+    public let teams: [String]?
```

Comment:
Wouldn't a non-optional empty array be more user friendly?

Reply:
That's actually what I've implemented first, to have parity between Android and iOS SDKs. However, I've learned `[]` is a valid `teams` field, so I wanted to differentiate between multi-tenancy enabled clients and disabled clients (by making these fields optional). They'll be `nil` in case backend doesn't send these fields, which means they don't have multi-tenancy enabled.
That being said, I'm still not sure if it should be optional or `[]`

Reply:
I don't know enough about this. My feeling is that even though it's not technically correct, using `[]` results in friendlier API. I don't think you need to know on the `User` model level if the client has multi-tenancy enabled.

Reply:
I agree with @VojtaStavik, app setting is the correct place to check for it.
Otherwise it creates another source of truth you need to maintain.

https://www.hyrumslaw.com/

Reply:
I've switched it to `[]` by default, instead of optional. I wonder if the same logic applies for `Channel.team`

Reply:
Yup, logic should be consistent.